Source/WebCore/platform/mac/WebPlaybackControlsManager.h

```diff
@@ -74,7 +74,8 @@ WEBCORE_EXPORT
 @property (getter=isPictureInPictureActive) BOOL pictureInPictureActive;
 @property BOOL canTogglePictureInPicture;
 - (void)togglePictureInPicture;
-- (void)toggleInWindow;
```

Comment:
Rather than remove this declaration, I'd recommend leaving it in place and implementing it in terms of the new enter/exit methods.